modules/library/jdbc/src/main/java/org/geotools/jdbc/MetadataTablePrimaryKeyFinder.java

```diff
@@ -62,7 +62,8 @@
     enum Policy {
         assigned,
         sequence,
-        autogenerated
+        autogenerated,
+        pfsequence
```

Comment:
What's the difference between ``sequence`` and ``pfsequence``.

Reply:
using `sequence` will have the insert statement include functional expression that will fetch next value from a sequence at time of insertion.

`pfsequence` will be used to generate number of next values on the given sequence before insertion is execute. So for example, if WFS-T call is insert 10 new features to underlying table/updatable view, if `pfsequence` is configured for the table/updatable view, the keyfetcher will fetch 10 atomic Ids from the sequence and treat them as attributes inside insert statement.

Reply:
Best to add javadocs explaining this difference (and document the others while at it).